admin/devurls.md

```diff
@@ -22,35 +22,40 @@ your cluster, you'll need to modify your:
 
 ### Step 1: Modify the Helm chart
 
-Set `devurls.host` to a wildcard domain pointing to your ingress controller:
+Set `devurls.host` to a wildcard domain:
 
 ```shell
-helm upgrade coder coder/coder --set devurls.host="*.my-custom-domain.io"
+helm upgrade coder coder/coder --set coderd.devurlsHost="*.my-custom-domain.io"
 ```
 
-If you're using the default ingress controller, specifying a value for
-`devurls.host` automatically adds a rule that routes dev URL requests to the
-user's workspace:
+**Note:** If you are providing an ingress controller, then you will need to add
+the rule manually to the ingress. If you are unsure, skip this step:
 
 ```yaml
  - host: "*.my-custom-domain.io"
     http:
       paths:
       - path: /
         backend:
-          serviceName: envproxy
+          serviceName: coderd
           servicePort: 8080
```

Comment:
@kylecarbs I'm unsure if this port is right and I don't believe we have any docs on providing an ingress. Should I omit this for now?